### PR TITLE
ARIA-399: Added Dockerfile, docker-compose and README to execute ARIA in a container environment

### PR DESCRIPTION
**Pull request by vishwanathj**

### CODE REVIEW

File: samples/CentOS/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
+RUN python get-pip.py
+
+RUN pip install --upgrade pip setuptools
+
+RUN pip install apache-ariatosca[ssh]==$ARIA_VER --no-binary apache-ariatosca
+
+WORKDIR /tmp
+
+RUN git clone https://github.com/apache/incubator-ariatosca.git /tmp/incubator-ariatosca
```
Comment (johnament) [inline samples/CentOS/Dockerfile:21]: We shouldn't encourage users to use master.  they should point to released code.

File: samples/CentOS/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM centos:7
```
Comment (johnament) [inline samples/CentOS/Dockerfile:1]: Missing standard license header.

File: samples/CentOS/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  aria:
+    #Make sure to provide the right value for image field below and uncomment the below line
+    #image: vishwanathj/aria_011:7
```
Comment (johnament) [inline samples/CentOS/docker-compose.yaml:5]: If we plan to distribute a docker image, perhaps that name should go here by default?

**Review by johnament — CHANGES_REQUESTED** [review]

File: samples/CentOS/Dockerfile (hunk 1 above)
Comment (tnadeauaria) [reply samples/CentOS/Dockerfile:21]: We have a plan to push released images as part of the release process for AriaTosca; this is just a quick initial push to help us get the kinks out and lay the ground work for that.

Comment (vishwanathj) [reply samples/CentOS/Dockerfile:21]: Please refer on L25 below that we are checking out the released tag version once the clone from repository has been done.

File: samples/CentOS/Dockerfile (hunk 2 above)
Comment (vishwanathj) [reply samples/CentOS/Dockerfile:1]: Would appreciate if you can share a link to the standard license header that needs to be used. Is it mandatory for Dockerfiles and docker-compose files as well?

File: samples/CentOS/docker-compose.yaml (hunk 3 above)
Comment (vishwanathj) [reply samples/CentOS/docker-compose.yaml:5]: Distributing docker images might be a future plan. It occurred  to me as well that this docker-compose file should be updated when we start publishing docker images to docker repo.

Comment (tnadeauaria) [reply samples/CentOS/docker-compose.yaml:5]: I haven't seen any more comments on this PR so I think we should be able to merge it in.

**Comment by vishwanathj** [comment]
Feedback was given by Tom N and Arthur to move this to the "incubator-ariatosca" repo, hence closing this PR.

**Pull request closed without merging**